### ISSUE TITLE
Status output for time-consuming operations

### ISSUE THREAD

**Issue by achilleas-k** [G-Node/gin-cli#24]
This isn't exactly like issue #7 (Progress Bars), but it is more about informing the user about what is happening, especially when an operation can take a while.

When working with very large files, some steps in an operation, like computing the hash when adding new files, can take a long time. A progress bar for data transfers wouldn't help here; there's a long computation occurring **before** the actual transfer begins. What we need is a message in the style of `Preparing files for upload...`, perhaps even a file count progress indicator.

Basically, we need anything that informs the user that *something is happening*, otherwise it looks like the program is hanging.
